src/Entities/Player.cpp

```diff
@@ -146,6 +146,22 @@ cPlayer::cPlayer(cClientHandlePtr a_Client, const AString & a_PlayerName) :
 
 
 
+void cPlayer::InitializePlayer()
```

Comment:
Reason for distinction? cEntity::Initialize also adds the entity to the chunk, is this done somewhere?